e2etest/bgp_test.go

```diff
@@ -70,6 +71,7 @@ type containerConfig struct {
 }
 
 var _ = ginkgo.Describe("BGP", func() {
+	var uc config.UpdateCleaner
```

Comment:
`configUpdater` ? uc is not really clear

Reply:
done